real_time/surface_src/java_src/Alice/src/test/java/TestWayPointFollowerPlanner.java

```diff
@@ -13,6 +16,8 @@
  */
 public class TestWayPointFollowerPlanner {
 
+	WayPointFollowerPlanner planner;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382738414/issues/source?bid=2938665&fileBranchId=3226464#l19)